cmd/environments/environments.go

```diff
@@ -25,6 +25,16 @@ func NewEnvironmentsCmd(
 				analytics.CmdRunEventProperties(cmd, "environments"),
 			)
 		},
+		PersistentPostRun: func(cmd *cobra.Command, args []string) {
+			analyticsTracker.SendEvent(
+				viper.GetString(cliflags.AccessTokenFlag),
+				viper.GetString(cliflags.BaseURIFlag),
+				"CLI Command Completed",
```

Comment:
Seems like we could make separate functions for `CLI Command Run` events and `CLI Command Completed` events so we don't have to type those strings out everywhere. Something like `SendCommandRunEvent()` and `SendCommandCompletedEvent()`.

Reply:
Yeah that sounds reasonable